server/public/plugin/client_rpc.go

```diff
@@ -683,6 +682,7 @@ func (s *apiRPCServer) PluginHTTPStream(args *Z_PluginHTTPStreamArgs, returns *Z
 
 			// Connect to response body stream and stream the response body
 			go func() {
+				defer r.Body.Close()
```

Comment:
Depending on how strict/safe you want to be, you might want to call Close in a few more places, in case the goroutine never runs due to the interface check failing or response being nil.

Reply:
Agreed we might as well be safe here.